packages/models/src/users.ts

```diff
@@ -105,7 +103,9 @@ export class Users {
     const rows = await knex<IUser.Row>(this.table)
       .select("*")
       .where(key, value);
-    return rows.map((row) => this.from(row));
+
+    const users = rows.map(row => this.from(row));
+    return users;
```

Comment:
It can be done in one line of code without affecting the clarity. 


```suggestion
    return rows.map(row => this.from(row));
```